oasis-client/oasis-client.go

```diff
@@ -219,7 +219,7 @@ func (oc *grpcOasisClient) GetStakingEvents(ctx context.Context, height int64) (
 	var gotBlkHash bool
 	var blkHash []byte
 	for i := range evts {
-		e := &evts[i]
+		e := evts[i]
```

Comment:
Since these are now pointers it should be fine if you just did `for i, e := range evts {` above.

Reply:
ok